src/style.css

```diff
@@ -1 +1,54 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector as it impacts performance. Set styles only for elements that require them. Zeroing out your margins, paddings, or other styles with '*' is still inefficient for the browser.